pkg/karmadactl/cmdinit/cmdinit.go

```diff
@@ -149,6 +152,7 @@ func NewCmdInit(parentCommand string) *cobra.Command {
 	flags.StringVar(&opts.ExternalEtcdKeyPrefix, "external-etcd-key-prefix", "", "The key prefix to be configured to kube-apiserver through --etcd-prefix.")
 	// karmada
 	flags.StringVar(&opts.CRDs, "crds", kubernetes.DefaultCrdURL, "Karmada crds resource.(local file e.g. --crds /root/crds.tar.gz)")
+	flags.StringVar(&opts.KarmadaInitFilePath, "init-config", "", "Karmada init file path")
```

Comment:
```suggestion
	flags.StringVar(&opts.KarmadaInitFilePath, "config", "", "Karmada init file path")
```
@liangyuanpeng I think `--config` should be fine, because its under `init`, so we can use a shorter name.
`karmadatl init --config`.

Reply:
oops, sorry about that, I confused it with `karmadactl config image xxx`, please help restore `init-config` to `config`
@tiansuo114

Reply:
OK,the changes in these areas have been completed, and now we are waiting for the GitHub CI process to run.